src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -207,9 +207,11 @@ public void FlushMetric()
         /// <returns></returns>
         public static string GenerateSha256HashString(string originInput)
```

Comment:
It is a public function, so the .Net guidelines say we should always do a null check